File: Server/models/Restaurant.js
const { v4: uuidv4 } = require('uuid');

class Restaurant {
    constructor(name, imageUrl, description, hours, foodItemIds) {
        this.name = name;
        this.imageUrl = imageUrl;
        this.description = description;
        this.hours = hours;
        this.foodItemIds = foodItemIds;
        this.id = uuidv4();
    }
}

module.exports = Restaurant;

File: Dyne/Dyne/Model/Restaurant.swift
//
//  Restaurant.swift
//  Dyne
//
//  Created by Malek Amiri on 10/10/20.
//  Copyright © 2020 Malek Amiri. All rights reserved.
//

import UIKit

class Restaurant {
    var name: String
    var imageUrl: String
    var description: String
    var hours: String
//    var foodItemIds
    var id: String
    var image: UIImage?
    
    init(name: String, imageUrl: String, description: String, hours: String, id: String) {
        self.name = name
        self.imageUrl = imageUrl
        self.description = description
        self.hours = hours
        self.id = id
    }
}
